src/assets/js/protocol/protocol-sidemenu.js

```diff
@@ -0,0 +1,30 @@
+(function() {
+    'use strict';
+
+    var menu = document.querySelector('.mzp-c-sidemenu');
+    var menuMain = document.querySelector('.mzp-c-sidemenu-main');
+    var menuToggle = document.querySelector('.mzp-js-toggle');
+    var menuExpanded;
```

Comment:
I don't think this variable needs to be this scope for any reason, you can probably just declare it inside the `click` even listener?